src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -97,6 +98,16 @@ public function delete(string $resource, string $id): void
         $this->request('DELETE', sprintf('/new-api/%s/%s', $resource, $id), []);
     }
 
+    public function applyTransition(string $resource, string $id, string $transition): void
+    {
+        $this->request(
+            'PATCH',
+            sprintf('/new-api/%s/%s/apply_transition/%s', $resource, $id, $transition),
```

Comment:
```suggestion
            sprintf('/new-api/%s/%s/apply-transition/%s', $resource, $id, $transition),
```
But $resource could have `_`, so do we have any convention?